qiita_db/util.py

```diff
@@ -863,7 +863,7 @@ def convert_to_id(value, table, text_col=None):
                                      % (value, table))
         # If there was a result it was a single row and and single value,
         # hence access to [0][0]
-        return _id[0][0]
+        return int(_id[0][0])
```

Comment:
I don't recall at the moment what was specifically raising the error -- I'll remove the change and run the tests to find out what was failing.